Resources/Prototypes/Entities/Virtual/virtual_item.yml

```diff
@@ -5,4 +5,5 @@
   noSpawn: true
   components:
   - type: Item
+    size: Ginormous # no storage insertion visuals
```

Comment:
What does this mean?

Reply:
The existing logic for drawing the item grid preview doesn't check action blocker (StorageContainer.cs line 341).

I could add that check, which would get run a lot in a FrameUpdate(). I thought that might be expensive, so setting the size to Ginormous achieves the same thing of hiding the grid preview.

I can switch to that solution if you prefer. I could also update that yml comment to be clearer.

![image](https://github.com/user-attachments/assets/1a33f2e3-be5e-46d8-ace4-e6c0d5aad871)

Reply:
I was almost tempted to say it should probably have its own thing but I also realised the effort is likely not worth the separation.